app/routes/events/components/EventList.js

```diff
@@ -128,6 +129,12 @@ const EventList = (props: EventListProps) => {
       <EventListGroup name="Neste uke" events={events.nextWeek} />
 
       <EventListGroup name="Senere" events={events.later} />
+
+      {!props.events && (
```

Comment:
Since `props.events` is always an Array of events, it will always evaluate to `true` - `Boolean([]) == true`. There are many ways to check if an array is empty, but using [isEmpty](https://lodash.com/docs/4.17.4#isEmpty) from lodash (javascript lib) is a nice and explicit way to do it. 
 
```jsx
import { isEmpty } from 'lodash';
[...]
{ isEmpty(props.events) && <EmptyState .... /> }
```

Reply:
Yes, I found out when changing the EventListGroup, but forgot to change it here, thanks :+1: